components/shooter.py

```diff
@@ -110,6 +110,8 @@ def __init__(self) -> None:
         self.inclinator_controller.setTolerance(ShooterComponent.INCLINATOR_TOLERANCE)
         SmartDashboard.putData(self.inclinator_controller)
 
+        self.desireStop = False
```

Comment:
nit: [use `snake_case` for variable names](https://wiki.thedropbears.org.au/sections/software/coding-standards#h.p_1tiHT35Sqjyr)

```suggestion
        self.desire_stop = False
```